docs/_docs/installation/ubuntu.md

```diff
@@ -24,7 +24,10 @@ source ~/.bashrc
 Finally, install Jekyll:
 
 ```sh
-gem install jekyll bundler
+gem install bundler
```

Comment:
```suggestion
gem install bundler jekyll
```

Reply:
@chocksmith Did you remove jekyll on purpose?

Reply:
Yes, I did. 
Look here: https://github.com/jekyll/jekyll/issues/5165#issuecomment-491240227

Tks!